quality_of_life_changer.client/src/redux-modules/user-profile/sagas.js

```diff
@@ -0,0 +1,25 @@
+import { put, takeEvery } from "redux-saga/effects";
+import { userProfileConstants } from "./constants";
+import * as actions from "./actions";
+import fetcher from "../../utils/fetcher";
+import { toastr } from "react-redux-toastr";
+
+function* fetchUserCalendars({ id }) {
+  try {
+    toastr.info("Wait server response");
+    const { data } = yield fetcher.get(
+      `"api/user/",${id},"/profile/calendars"`
```

Comment:
super weird string interpolation.
`api/user/${id}/profile/calendars`  in backticks- this works.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals